src/main/java/com/netgrif/application/engine/pdf/generator/service/PdfDataHelper.java

```diff
@@ -152,6 +165,12 @@ protected void generateField(DataGroup dataGroup, LocalisedField field) {
                     pdfField = createMultiChoiceField(dataGroup, (LocalisedMultichoiceField) field);
                     pdfFields.add(pdfField);
                     break;
+                case I18N:
+                    if (field.getComponent() != null && Objects.equals(field.getComponent().getName(), "divider")) {
```

Comment:
would be better if the string constant was a real constant and not a value literal

Reply:
Done.